django_airavata/apps/workspace/static/django_airavata_workspace/js/components/experiment/ExperimentEditor.vue

```diff
@@ -166,7 +167,8 @@ export default {
     return {
       projects: [],
       localExperiment: this.experiment.clone(),
-      invalidInputs: []
+      invalidInputs: [],
+      input_name:''
```

Comment:
This isn't used, and in fact, all of the changes to this file are unnecessary as far as I can tell.